package.json

```diff
@@ -78,13 +79,14 @@
     "gulp-util": "~3.0.7",
     "husky": "^0.12.0",
     "karma": "^1.0.0",
-    "karma-chrome-launcher": "^2.2.0",
+    "karma-chrome-launcher": "^2.0.0",
```

Comment:
Seems that your code is based on our older code. This line should be kept `^2.2.0`.